.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: ci
+
+on: [push]
+
+jobs:
+  build-and-lint:
```

Comment:
we should probably change this name to be more representative. Could be something more vague like "code-checks"